CMakeLists.txt

```diff
@@ -6,6 +6,15 @@ project(decomp2d
 set(version 2.0.0)
 enable_testing()
 
+option(DOUBLE_PRECISION "Build 2decomp_fft with double precision" ON)
+if (DOUBLE_PRECISION)
+  add_definitions("-DDOUBLE_PREC")
```

Comment:
This duplicates https://github.com/2decomp-fft/2decomp-fft/blob/main/cmake/D2D_Compilers.cmake#L93 ?

Should be `target_compile_definitions(decomp2d PUBLIC -DADIOS2)` ?

This question was raised in https://github.com/2decomp-fft/2decomp-fft/pull/195

Reply:
Looks like a merge error, will fix.

Reply:
Done